addons/im_livechat/models/res_users.py

```diff
@@ -11,6 +11,8 @@ class Users(models.Model):
     _inherit = 'res.users'
 
     livechat_username = fields.Char("Livechat Username", help="This username will be used as your name in the livechat channels.")
+    # Discuss category livechat open status
+    is_category_livechat_open = fields.Boolean("Is category livechat open", default=True)
```

Comment:
We have 2M users in odoo... we don't want to add a new column with True by default. Actually we don't want any new column, but if really necessary, should be False for most users.